src/soar_sdk/params.py

```diff
@@ -132,3 +132,37 @@ def _to_json_schema(cls) -> dict[str, InputFieldSpecification]:
             params[field_name] = params_field
 
         return params
+
+
+class OnPollParams(Params):
+    """
+    Parameters for on_poll.
+
+    Used to define the specific parameters for the on_poll action.
+    """
+
+    start_time: int = Param(
+        description="Start of time range, in epoch time (milliseconds).",
+        required=False,
+    )
+
+    end_time: int = Param(
+        description="End of time range, in epoch time (milliseconds).",
+        required=False,
+    )
+
+    container_count: int = Param(
+        description="Maximum number of container records to query for.",
+        required=False,
+    )
+
+    artifact_count: int = Param(
+        description="Maximum number of artifact records to query for.",
+        required=False,
+    )
+
+    container_id: str = Param(
+        description="Container IDs to limit the ingestion to.",
+        required=False,
+        allow_list=True,
+    )
```

Comment:
Is this supposed to be a comma-separated list of container IDs?

Reply:
Yeah, can be a comma-separated list of container IDs. I just used the description from the official docs on on_poll but should we update to be more explicit?

Reply:
Yeah let's be more explicit here
```suggestion
    container_id: str = Param(
        description="Comma-separated list of container IDs to limit the ingestion to.",
        required=False,
        allow_list=True,
    )
```